include/natalie/module_object.hpp

```diff
@@ -85,7 +85,7 @@ class ModuleObject : public Object {
 
     void included_modules(Env *, ArrayObject *);
     Value included_modules(Env *);
-    Vector<ModuleObject *> included_modules() { return m_included_modules; }
+    Vector<ModuleObject *> &included_modules() { return m_included_modules; }
```

Comment:
This now returns a mutual reference, so this code:
```c++
module->included_modules().clear()
```
now resets the included modules, where before it operated on it's own copy.

I think this one should be `const` instead.